shadowsocks-csharp/Data/zh_CN.txt

```diff
@@ -183,6 +183,7 @@ Proxy handshake failed=代理握手失败
 Register hotkey failed=注册快捷键失败
 Cannot parse hotkey: {0}=解析快捷键失败: {0}
 Timeout is invalid, it should not exceed {0}=超时无效，不应超过 {0}
+Cannot find the plugin program file=找不到插件程序文件

```

Comment:
Add program name here?

Reply:
File name will be mentioned in the exception it self in `Sip003Plugin.cs`
```
throw new FileNotFoundException(I18N.GetString("Cannot find the plugin program file"), _pluginProcess.StartInfo.FileName, ex);
```

![image](https://user-images.githubusercontent.com/3608762/70857419-b2f86e80-1f28-11ea-829d-d0b91f159ff0.png)